### ISSUE TITLE
Blender animation suggestion

### ISSUE THREAD

**Issue by emackey** [KhronosGroup/glTF-Render-Fidelity#6]
I noticed in [blender-cycles-golden.png](https://github.com/KhronosGroup/glTF-Render-Fidelity/blob/main/test/goldens/khronos-AnimatedTriangle/blender-cycles-golden.png) that the animated triangle is not at the start of its animation cycle.

Blender uses a default start frame number of "1", but glTF animations start at time zero (in seconds, not frame numbers).  The answer here might be to add some Python commands to the blender render script:

```
bpy.context.scene.frame_start = 0
bpy.context.scene.frame_current = 0
```

This shouldn't alter non-animated glTF examples, and should make the animated ones start at the beginning, like other engines.

**Comment by elalish** [comment]
Excellent, want to make a PR?

**Comment by emackey** [comment]
Certainly, but, I'm blocked by #4 because I would need to regenerate goldens.
